### ISSUE TITLE
SVG href links with mtextInheritFont:true are misplaced

### ISSUE THREAD

**Issue by dpvc** [mathjax/MathJax#2588]
> Also, note that there are some bugs with `mtextInheritFont: true` in v3 in both CommonHTML and SVG output. See #2257 and #2189.

> `mtextInheritFont: true` in v3 in SVG output also upsets the scrolling to labelled equations, targeting the baseline instead of the top of the equation. (This doesn't happen in CHTML output.)

> For example, add this option to the [equation-refs demo](https://mathjax.github.io/MathJax-demos-web/equation-refs.html) and change the output to SVG, then click on a link to `(1)`: only the lower part of the equation is visible (using Safari and Firefox).

_Originally posted by @pdmosses in https://github.com/mathjax/MathJax/issues/2329#issuecomment-748855560_

**Comment by dpvc** [comment]
This seems to affect Safari only when `mtextInheritFont` is `true`, but affects Firefox all the time.

I suspect it is due to the transform that inverts the *y*-coordinates not being handled properly, but haven't verified that.   SVG usually has the *y*-axis pointing down rather than up, so height would be negative and depth positive.  We may eliminate that transform in the font update in 2021.

**Comment by pdmosses** [comment]
Thanks for this. (I originally intended to submit it as a separate issue, but when I noticed your comment in #2329 about bugs with `mtextInheritFont: true` in v3, I thought it might be related.)

I'm now using `\smash{\raise{2.4ex}{\cssId{#1}{}}}` to make targets of href links visible, following a [suggestion](https://tex.stackexchange.com/a/197244) for addressing a similar issue with `hyperref`.  It's not a general solution, but it works well enough for me in Firefox.
